mkdocs.yml

```diff
@@ -8826,9 +8825,9 @@ plugins:
 - search
 - exclude-search:
     exclude_unreferenced: true
-- git-revision-date-localized:
-    enable_creation_date: true
-    type: timeago
+#- git-revision-date-localized:
```

Comment:
Why do you want to remove this ?

Reply:
Sorry, committed by mistake. I comment this out when I am testing doc changes locally before push. Otherwise it takes a long time to deploy locally. And just before committing, I uncomment these lines.
Forgot to do it this time. 

I'll uncomment and commit again.